aea/decision_maker/messages/transaction.py

```diff
@@ -39,55 +39,52 @@ class TransactionMessage(InternalMessage):
     class Performative(Enum):
         """Transaction performative."""
 
-        PROPOSE = "propose"
-        SIGN = "sign"
-        ACCEPT = "accept"
-        REJECT = "reject"
+        PROPOSE_FOR_SETTLEMENT = "propose_for_settlement"
+        SUCCESSFUL_SETTLEMENT = "successful_settlement"
+        FAILED_SETTLEMENT = "failed_settlement"
+        REJECTED_SETTLEMENT = "rejected_settlement"
+        PROPOSE_FOR_SIGNING = "propose_for_signing"
+        SUCCESSFUL_SIGNING = "successful_signing"
+        REJECTED_SIGNING = "rejected_signing"
```

Comment:
Would be useful at some point to have documentation and illustration (e.g. with sequence diagrams) about the possible usages of the different performatives.

Reply:
Yes agreed. Opening an issue.